cmd/statusd/main.go

```diff
@@ -150,6 +155,50 @@ func main() {
 		return
 	}
 
+	if config.PushNotificationServerConfig.Enabled {
+		if config.NodeKey == "" {
+			logger.Error("node key needs to be set if running a push notification server")
+			return
+		}
+
+		identity, err := crypto.HexToECDSA(config.NodeKey)
+		if err != nil {
+			logger.Error("node key is invalid", "error", err)
+			return
+		}
+
+		// Generate installationID from public key, so it's always the same
+		installationID, err := uuid.FromBytes(crypto.CompressPubkey(&identity.PublicKey)[:16])
+		if err != nil {
+			logger.Error("cannot create installation id", "error", err)
+			return
+		}
+
+		db, err := appdatabase.InitializeDB(config.DataDir+"/"+installationID.String()+".db", "")
+		if err != nil {
+			logger.Error("failed to initialize app db", "error", err)
+			return
+		}
+
+		options := []protocol.Option{
+			protocol.WithPushNotifications(),
+			protocol.WithPushNotificationServerConfig(&config.PushNotificationServerConfig),
+			protocol.WithDatabase(db),
+		}
+
+		messenger, err := protocol.NewMessenger(identity, gethbridge.NewNodeBridge(backend.StatusNode().GethNode()), installationID.String(), options...)
+		if err != nil {
+			logger.Error("failed to create messenger", "error", err)
+			return
+		}
+
+		err = messenger.Start()
```

Comment:
Okay, we start the messenger, but where do we start the norifications server?

Reply:
That's based on the config, the messenger will start it if passed `PushNotificationServer` -> `Enabled` on start

Reply:
I see, could we add a comment here making that clear?

Reply:
done